### ISSUE TITLE
Token utilities and characteritics specs organization

### ISSUE THREAD

**Issue by ivan-calderon** [jpmorganchase/salt-ds#4139]
Ongoing discussions about if the [token tile sticker](https://www.figma.com/design/i8kSFpGbHIWXnv6LFvWKdU/Salt-(Next)-Spec-Utilities?m=auto&node-id=4722-2592&t=nRBVcwKVaHcTrWDs-1) utility should organize tokens by characteristics, attributes or be element specific. Some reference info below:
- Organization by attributes (foreground, background, border)
- Organization by element (button, input, helper text)
- Organization by characteristics CURRENT (Container, content, actionable, editable) 

c.c. @navkaur76, @origami-z

**Comment by pseys** [comment]
Components for each characteristic have been added in a branch, but not yet merged. Each component contains an instance for each token with props for each token value group e.g. variant, attribute, state.

**Comment by bhoppers2008** [comment]
@pseys @ivan-calderon is this issue complete?

**Comment by pseys** [comment]
@bhoppers2008 Yes and no! We made the change based on Salt (Next) Styles library. I'm not sure if @ivan-calderon has updated the file to use Salt DS Themes variables yet or whether that still needs to be done.

**Comment by bhoppers2008** [comment]
Ok. Please update or close as required once you've discussed.

**Comment by pseys** [comment]
@ivan-calderon lets raise this in the next design share – it would be good to re-familiarise the team with it and agree on whether its needed or not.
